documentation/github_feature_request_template.md

```diff
@@ -0,0 +1,52 @@
+# Feature Request Template
+
+## **Title**
+Provide a clear and descriptive title for the feature request.
+
+## **Description**
+A detailed description of the feature request, including the problem it aims to solve and the benefits it will bring.
+
+## **Motivation**
+Explain why this feature is necessary and how it will improve the project.
+
+## **Proposed Solution**
+Describe the proposed solution in detail. Include any relevant code snippets, diagrams, or mockups.
+
+## **Alternatives Considered**
+List any alternative solutions or approaches that were considered and explain why they were not chosen.
+
+## **Additional Context**
+Add any other context or information that might be useful, such as related issues, links to discussions, or references to external resources.
+
+## **Checklist**
+- [ ] I have searched for existing feature requests that might be related to this one.
+- [ ] I have provided a detailed description of the feature request.
+- [ ] I have included all necessary information about the motivation and proposed solution.
+- [ ] I have attached relevant diagrams, mockups, or code snippets.
+
+## **Labels**
+Assign appropriate labels to categorize the feature request (e.g., enhancement, new feature).
+
+## **Assignees**
+Assign the feature request to relevant team members.
+
+## **Milestones**
+- [ ] Task 1
+- [ ] Task 2
+- [ ] Task 3
+
+## **Linked Issues**
+Link to any related issues or pull requests.
+
+## **Error Handling & Logging**
+- Ensure that any potential errors related to the feature are considered and documented.
+- Include logging requirements if applicable.
+
+## **README Generation**
+- Ensure that the features are documented in the project's relevant README file.
+
```

Comment:
Also something like this:

## **ADR  Generation**
- Ensure that all decisions that manifest as part of the feature implementations get appropriately documented as one or more ADRs

Reply:
Added

Reply:
nice